src/components/ContractCardForContractor/index.jsx

```diff
@@ -219,6 +220,17 @@ export const ContractCardForContractor = ({
         </Grid.Column>
       )}
 
+      {displayModal && (
+        <ConfirmationMessage
+          description="At this stage we notice that you will not able to change contract status after delivering result. <br><br>If customer decides to decline your contract, you will not be paid."
```

Comment:
What do you think if we pass `description` as a `children` like:
```
<ConfirmationMessage {...props}>
  At this stage we notice...
</ConfirmationMessage>
```

It would be much better, isn't it?

Reply:
Sure, I totally agree with you :)